File: go/model/Period.go
package model

type Period struct {
	Id         int64  `db:"id" json:"id"`
	HappenTime string `db:"happen_time" json:"happenTime"`
	Type       int    `db:"type" json:"type"`
	CreateTime string `db:"create_time" json:"createTime"`
	UpdateTime string `db:"update_time" json:"updateTime"`
}

File: vue/daily-console/src/network/period.js
import request from './request'

export function PeriodDto(){
    this.id=0,
    this.type=0,
    this.happenTime=''
}

export function getLastRecord(){
    return request({
        url:'/period/getLastRecord',
        method:'GET'
    })
}

export function saveRecord(period){
    console.log(period)
    console.log(JSON.stringify(period))
    return request({
        url:'/period/saveRecord',
        method:'POST',
        data:JSON.stringify(period)
    })
}

export function listRecord(pageDto){
    return request({
        url:'/period/listRecord',
        method:'POST',
        data:JSON.stringify(pageDto)
    })
}
